### ISSUE TITLE
More balanced versions

### ISSUE THREAD

**Issue by Tenebral228** [Derec-Mods/Mace-Mod#14]
Minecraft Version: 1.12-1.19.2
Mod Version: 😅 latest?
Hello, I would like to see versions of the mod with new balance changes (especially for 1.16.5).
Sorry for the inconvenience and (maybe) impossible demands.
Have a great day!

**Comment by DereC4** [comment]
im open to making a "balanced version" that's an optional download. I'll look into it

**Comment by Tenebral228** [comment]
Thanks a lot!

**Comment by DereC4** [comment]
first balance release coming out today test it on 1.20.1

**Comment by Tenebral228** [comment]
> first balance release coming out today test it on 1.20.1

Thanks a lot! I'm glad that you are ready to listen to players, even such uncommon ones.

**Comment by Tenebral228** [comment]
Hello, I don't know if you know about extra files, but to make it easier to add balanced versions, you can add them to extra files for the main files of your mod, or you can change their development stage to beta version (and declare that beta files are not laggy, but are balanced versions of mod). (translated)

**Comment by DereC4** [comment]
ah sounds good i'll look into that later but ty that would help clear things up

**Comment by Tom28281** [comment]
Please do that by the way, forge thinks it's outdated and makes me download the balanced version when updating.

**Comment by DereC4** [comment]
@FrAmed46 @Tenebral228 Haven't checked in a while, is anything an issue as of today?

**Comment by Tenebral228** [comment]
> [@FrAmed46](https://github.com/FrAmed46) [@Tenebral228](https://github.com/Tenebral228) Haven't checked in a while, is anything an issue as of

> [@FrAmed46](https://github.com/FrAmed46) [@Tenebral228](https://github.com/Tenebral228) Haven't checked in a while, is anything an issue as of today?

No, but the attack speed of the mace should be slower in my opinion (0.6), all the exact characteristics can be found on the minecraft wiki: https://minecraft.wiki/w/Mace
